planner/core/find_best_task.go

```diff
@@ -1982,9 +1983,11 @@ func (ds *DataSource) convertToPointGet(prop *property.PhysicalProperty, candida
 		pointGetPlan.IndexValues = candidate.path.Ranges[0].LowVal
 		pointGetPlan.PartitionInfo = partitionInfo
 		if candidate.path.IsSingleScan {
-			cost = pointGetPlan.GetCost(candidate.path.IdxCols)
+			pointGetPlan.accessCols = candidate.path.IdxCols
+			cost = pointGetPlan.GetCost()
```

Comment:
The same logic in both if/else branch, please move it out to make it clear.

Reply:
Fixed, PTAL